src/Microsoft.Azure.WebJobs.Host/Queues/QueueProcessorFactoryContext.cs

```diff
@@ -32,17 +35,19 @@ public QueueProcessorFactoryContext(CloudQueue queue, TraceWriter trace, CloudQu
             Queue = queue;
             PoisonQueue = poisonQueue;
             Trace = trace;
+            Logger = loggerFactory?.CreateLogger(LoggingCategories.CreateTriggerCategory("Queue"));
```

Comment:
What is the benefit of scoping these logs to "Queue"? Also, I don't see you doing this for other triggers like blob/servicebus?